lib/modules/apostrophe-seo-doc-type-manager/public/js/editor-modal.js

```diff
@@ -0,0 +1,39 @@
+$(function() {
+  var superPopulate = apos.schemas.fieldTypes.string.populate;
+  apos.schemas.fieldTypes.string.populate = function(data, name, $field, $el, field, callback) {
+    return superPopulate(data, name, $field, $el, field, function(err) {
+      if (err) {
+        return callback(err);
+      }
+      console.log('hello');
+      if (field.seoSoftMin || field.seoSoftMax) {
+        $field.on('textchange', update);
+        $field.on('change', update);
+        function update() {
+          var len = $field.val().length;
+          if (len && (len < field.seoSoftMin)) {
+            warning('Fewer than ' + field.seoSoftMin + ' characters');
```

Comment:
The messages could be clearer. How about "You have exceed the recommended maxium 300 characters," and "You have not reached the recommended minimum of 60 characters." They're longer, but less ambiguous. Since the yellow isn't screaming "I'm broken" to me more clarity seems important.